### DISCUSSION TITLE
Add ability to change other settings in the application based profiles

### DISCUSSION THREAD

**Discussion by HahBikes** [Valkirie/HandheldCompanion#320] [Ideas]
Currently, we can change the TDP and the UMC settings in the profiles section.

It would be great If we can change the newly added refresh rate, screen resolution, brightness as well as volume in the profile section too (_so that we can have separate settings based on the application running_).

**Comment by pparmy** [comment]
As well as the option to have a preset for lizard mode on profiles! Or maybe forcing Steam to close, or disable inputs, while a certain profile .exe file is running.

**Comment by CasperH2O** [comment]
Several of the above mentioned items are now per application i.e. profile based:
- TDP -> part of power presets
- UMC -> part of layout
- refresh rate -> part of graphics settings

Lizard mode was removed @pparmy
